packages/commonwealth/server/migrations/20250303120000-fix-quest-model.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.sequelize.query(
+        `
+        -- create "artificial" system quest to award xp points on referral and address linking events
+        INSERT INTO "Quests" (
+          "id",
+          "name",
+          "description",
+          "image_url",
+          "start_date",
+          "end_date",
+          "created_at",
+          "updated_at")
+        VALUES (
+          -1,
```

Comment:
We just going to use negative numbers for all system quests? E.g. I have an open PR to award XP when a user joins Discord or follows on Twitter. Do I need to create a new system quest for that?

Reply:
It depends, if the action is configurable through regular quests (UI) or automatic like for these events where the user is not a real common user yet. In the case of joining via Discord, probably another system action... the twitter follow not sure, it can be configurable for existing users I'm guessing